README.md

```diff
@@ -1,4 +1,7 @@
-# Confita
+[](https://user-images.githubusercontent.com/4570448/40786144-a8e8ce34-64ea-11e8-90b9-cb838cd13b88.png)
+
+<img height="220" src="https://user-images.githubusercontent.com/4570448/40786144-a8e8ce34-64ea-11e8-90b9-cb838cd13b88.png">
+
```

Comment:
🌷 You should maybe version the logo in the repo and use relative links

Reply:
The image is referenced in a [closed issue](https://github.com/heetch/confita/issues/27). It's a common way to do like this.

Reply:
Actually this is an old hack, you create an issue, upload an image in it and close the issue. You normally have the guarantee the link doesn't get removed because it's used within an issue.
That's a good hack when the project is young

Reply:
I thought you guys had some high standards, and wouldn't rely on some old hacks to do the job 🤣 Good to know though